### PR TITLE
Release v202211

### PR DESCRIPTION
**Pull request by MakisH**
This is a rather large release with many new tutorials and tutorial cases. See the `CHANGELOG.md` for more.

### CODE REVIEW

**Comment by IshaanDesai** [comment]
TODOs:

- The [channel flow with chemical reaction tutorial](https://github.com/precice/tutorials/tree/develop/channel-transport-reaction) does not have a website page currently, so it needs to be added. I am doing this.
- Website link to [flow-over-heated-plate-two-meshes](https://github.com/precice/tutorials/tree/develop/flow-over-heated-plate-two-meshes) is broken and needs to be fixed.

@IshaanDesai is doing these.

**Comment by MakisH** [comment]
@MakisH remember to add links to the new tutorials that just got merged to the changelog

**Comment by kanishkbh** [comment]
I ran the following cases and they were successful:
- Flow over heated plate: solid-fenics + fluid-openfoam

- Channel Transport Reaction: chemical-fenics + fluid-fenics
    - chemical-fenics outputs 100 files, while fluid-fenics outputs 5000 files. Would be nice to document this temporal mismatch in the [tutorial documentation](https://precice.org/tutorials-channel-transport-reaction.html) (@boris-martin ).

**Comment by thesamriel** [comment]
Tested the following cases
- [x] channel-transport
- [x] oscillator (Change README instructions from `./run.sh -m` to `./run.sh -l` and `./run.sh -n` to `./run.sh -r`
- [x] partitioned-heat-condution-direct

**Review by IshaanDesai — APPROVED** [review]
I ran the following tutorials and checked the results for their correctness in ParaView:

- partitioned-heat-conduction: fenics-fenics, nutils-nutils
- perpendicular-flap: fluid-openfoam and solid-fenics, fluid-su2 and solid-calculix
- flow-over-heated-plate: fluid- openfoam and solid-nutils
- elastic-tube-3d: fluid-openfoam and solid-calculix

All cases were run the preCICE VM with the develop branch of the tutorials cloned manually.

**Comment by fsimonis** [comment]
Testing also in vm https://github.com/precice/vm/pull/50

**Comment by KyleDavisSA** [comment]
I tested the following OpenFOAM-CalculiX tutorials:
 * elastic-tube-3D - serial and parallel OF,
 * flow-over-heated-plate-two-meshes,
 * heat-exchanger, 
 * heat-exchanger-simplified,
 * partitioned-elastic-beam,
 * perpendicular-flap.

All test case ran perfectly fine with:
 * OpenFOAM v2012 with openfoam-adapter v1.2.0 from master branch ([7afeaed](https://github.com/precice/openfoam-adapter/commit/7afeaede9d9aec8d48457c77e8be9c6131897f44)),
 * CalculX V2.20 release
 * preCICE v2.5.0 release,
 * tutorials develop ([fc32d55](https://github.com/precice/tutorials/commit/fc32d55a458d5c9f84280677978c265d70b48c7b) )

**Comment by IshaanDesai** [comment]
> channel-transport/fluid-nutils fails. @thesamriel did you test using a different nutils version?

The code is indeed working only for Nutils v7.0 and up and the VM has Nutils v6.3. Updating the Nutils version in the VM is a solution.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,30 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+- Added a gitignore for code and binaries [#290](https://github.com/precice/tutorials/pull/290).
+- Added more documentation [#264](https://github.com/precice/tutorials/pull/264), [#265](https://github.com/precice/tutorials/pull/265), [#266](https://github.com/precice/tutorials/pull/266) and fixed some typos [#285](https://github.com/precice/tutorials/pull/285).
+
+### Changed
+
+- Adapted the multiple-perpendicular-flaps tutorial to a [new naming convention](https://precice.org/community-contribute-to-precice.html#contributing-tutorials) for tutorials with multiple participants [#303](https://github.com/precice/tutorials/pull/303).
+- Adapted the material properties of the FEniCS case in elastic-tube-3d to match the CalculiX case results [#259](https://github.com/precice/tutorials/issues/259).
+- Adapted the solid case of the Quickstart tutorial to work with the Intel oneAPI C++ compiler [#291](https://github.com/precice/tutorials/pull/291).
+- Updated the Nutils cases [#295](https://github.com/precice/tutorials/pull/295), [#298](https://github.com/precice/tutorials/pull/298).
```
Comment (uekerman) [inline CHANGELOG.md:32]:
```suggestion
- Made the Nutils case of the partitioned heat conduction tutorial compatible with Nutils versions 6 to 8 [#295](https://github.com/precice/tutorials/pull/295), [#298](https://github.com/precice/tutorials/pull/298).
```

@IshaanDesai Correct? The other Nutils cases do not yet have this compatibility, right?

**Review by uekerman — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (IshaanDesai) [reply CHANGELOG.md:32]: True, @MakisH just pointed out that the Nutils fluid solver in `perpendicular-flap/fluid-nutils` does not work with Nutilv v7.0 and needs v6.3. I am looking into whether updating all the cases is feasible.

**Comment by MakisH** [comment]
Thank you all very much for helping with testing! I also tested several cases myself over the past weeks, and I checked all of them in the VM, fixing some VM issues.

**Pull request merged**